### ISSUE TITLE
Missing docblocks

### ISSUE THREAD

**Issue by waltertamboer** [phpmath/vector#9]
All code files are missing docblocks.

**Comment by waltertamboer** [comment]
Should be in the form of:

``` php
<?php
/**
 * This file is part of phpmath/vector. (https://github.com/phpmath/vector)
 *
 * @link https://github.com/phpmath/vector for the canonical source repository
 * @copyright Copyright (c) 2015-2016 phpmath. (https://github.com/phpmath/)
 * @license https://raw.githubusercontent.com/phpmath/vector/master/LICENSE.md MIT
 */

namespace PHP\Math\Vector;
```

**Comment by waltertamboer** [comment]
Fixed in PR #12
